.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2.1
+
+jobs:
+  publish:
+    docker:
+      - image: cimg/openjdk:8.0-browsers
```

Comment:
We'll have to go through some trial & error process here (e.g. correctness, resource size, etc.).